qa/rpc-tests/test_framework/util.py

```diff
@@ -36,10 +36,13 @@ def sync_blocks(rpc_connections, wait=1):
     """
     Wait until everybody has the same block count
     """
+    t = 0
     while True:
         counts = [ x.getblockcount() for x in rpc_connections ]
+        print "   Syncing...Node Block Counts ==> " + str(counts) + ' Elapsed Time (s): ' + str(t)
         if counts == [ counts[0] ]*len(counts):
             break
+        t += wait
```

Comment:
Actually it's pretty useful to see where the scripts hang when/if the 
nodes don't sync...otherwise the test just hangs there and then proceeds 
10 or 15 minutes later without the user ever knowing there was a problem.



On 21/08/2015 6:27 AM, Jonas Schnelli wrote:
>
> In qa/rpc-tests/test_framework/util.py 
> <https://github.com/bitcoin/bitcoin/pull/6548#discussion_r37633247>:
>
> >          if counts == [ counts[0] ]*len(counts):
> >              break
> > +        t += wait
>
> Mind removing the "syncing..." log prints? Seems that this has nothing 
> to do with running on window?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/bitcoin/bitcoin/pull/6548/files#r37633247>.
>